baselines/tamuna/pyproject.toml

```diff
@@ -0,0 +1,141 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "tamuna"
+version = "1.0.0"
+description = "TAMUNA: Doubly Accelerated Federated Learning with Local Training, Compression, and Partial Participation"
+license = "Apache-2.0"
+authors = ["Ivan Agarský <agarskyivan@gmail.com>, Grigory Malinovsky"]
```

Comment:
The tests fail (on the CI) because the expected format is "name <email>". and comma separated. Eg. `["Ivan Agarský <agarskyivan@gmail.com>", "Grigory Malinovsky <email@gmail.com>"]`